docs/src/components/Sandbox/Preview/CollectStyles/CollectStyles.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import axios from 'axios';
+import urlJoin from 'url-join';
+
+const collectStyleContent = baseHref => {
+  const styleNodes = document.querySelectorAll('style, link[rel="stylesheet"]');
+
+  const sheetPromises = Array.from(styleNodes).map(el => {
+    if (el.nodeName === 'STYLE') {
+      return Promise.resolve(el.innerHTML);
+    }
+
+    const href = el.getAttribute('href');
+    const url = urlJoin(baseHref, href);
+
+    return axios.get(url)
```

Comment:
I'd just use the fetch api here instead of introducing a new dependency. For your use case, it's almost the same code.

Reply:
Does the project already have the pollyfill?

Reply:
Why would you need polyfill? This only runs in the browser right?

Reply:
More reason why axios would be a good choice no? As fetch still has gaps in support right?

Reply:
Only IE11 is out. As this is a docs page I think we don't need to worry about that.

Reply:
IE11 is definitely a crap browser granted, but I'd only consider dropping it if there was a deficit due to the support cost. Otherwise, I wouldn't intentionally limit my audience for the cost of a simple polyfill / alternative.

Reply:
Just checked. This page is completely broken in IE11. No surprise lol

Reply:
Just swapped it out for native fetch. We can revisit later if we need to.